docs/creating-a-custom-bot.md

```diff
@@ -0,0 +1,27 @@
+# Creating a custom bot
```

Comment:
Would be nice if this file is in the main README.md file. But make the change at own will.